docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+const variable = allVariables[0]; // variable being checked is first variable in template
+Openlaw.showInForm(variable, executionResult.executionResult);
+```
+
+**Response**
+
+Returns `true` if variable is of type that should be displayed in the input form in draft mode (e.g., [input variables](/markup-language/#variables) like Text, Number, Date, Address, etc.).
+
+### getType
+
+Get variable type.
+
+```scala
+getType(
+  variable: VariableDefinition
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `variable` | [`VariableDefinition`](#variabledefinition) | **Required.** A [`VariableDefinition` object](#variabledefinition) representing the variable. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const allVariables = Openlaw.getVariables(executionResult.executionResult, {});
+allVariables.forEach(variable => {
+  Openlaw.getType(variable));
+});
+```
+
+**Response**
+
+Returns variable type as a string.
+
+Example
+
+```js
+"Text"
+"Date"
+"YesNo"
+"Number"
+```
+
+### getName
+
+Get name of a variable.
+
+```scala
+getName(
+  variable: VariableDefinition
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `variable` | [`VariableDefinition`](#variabledefinition) | **Required.** A [`VariableDefinition` object](#variabledefinition) representing the variable. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const allVariables = Openlaw.getVariables(executionResult.executionResult, {});
+allVariables.forEach(variable => {
+  Openlaw.getName(variable));
+});
+```
+
+**Response**
+
+Returns name of variable as a string.
+
+Example
+
+```js
+"Company Name"
+"Effective Date"
+"No Services"
+"Number of Shares"
+```
+
+### getDescription
+
+Get description of a variable.
+
+```scala
+getDescription(
+  variable: VariableDefinition
+): String
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `variable` | [`VariableDefinition`](#variabledefinition) | **Required.** A [`VariableDefinition` object](#variabledefinition) representing the variable. |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const executionResult = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+const allVariables = Openlaw.getVariables(executionResult.executionResult, {});
+allVariables.forEach(variable => {
+  Openlaw.getDescription(variable));
+});
+```
+
+**Response**
+
+Returns description of variable as a string. The default description is the variable name. You can also modify the description that appears in the form as described in our [markup language documentation](/markup-language/#variables).
```

Comment:
The default description is the variable name. You can also modify the description that appears in the form as described in our [markup language documentation](/markup-language/#variables). -> I think it would be clearer if this was included after "Get description of a variable" above.

Reply:
I agree. I moved the explanation up to right after "Get description of a variable." under the heading.